cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -611,18 +634,18 @@ def generate_products_url_from_par_dict(self, products_url, par_dict) -> str:
         return request_url
 
     def generate_products_url_from_file(self, products_url, session_id, job_id) -> str:
-        job_monitor_status_json_file = f'scratch_sid_{session_id}_jid_{job_id}/query_output.json'
+        query_output_json_file = f'scratch_sid_{session_id}_jid_{job_id}/query_output.json'
         # to be handled now, with the job_id generated taking into account only the user_id
-        job_monitor_status_json_file_aliased = f'scratch_sid_{session_id}_jid_{job_id}_aliased/query_output.json'
+        query_output_json_file_aliased = f'scratch_sid_{session_id}_jid_{job_id}_aliased/query_output.json'
         request_url = ""
         file = None
         if self.scratch_dir:
             file = open(self.scratch_dir + '/query_output.json')
         else:
-            if os.path.exists(job_monitor_status_json_file_aliased):
-                file = open(job_monitor_status_json_file_aliased)
-            elif os.path.exists(job_monitor_status_json_file):
-                file = open(job_monitor_status_json_file)
+            if os.path.exists(query_output_json_file_aliased):
```

Comment:
so, 645-648 not covered by test, codecov complains? scratch existed?

Reply:
probably this check is not really necessary

Reply:
the scratch directory is always set